jekyll/_docs/docker.md

```diff
@@ -274,6 +274,38 @@ Please don't hesitate to [contact us](mailto:support@circleci.com)
 if you have any questions at all about how to best utilize Docker on
 CircleCI.
 
+#### Benefitting from test summaries:
+
+In order to view test summaries, you will need to utilize the docker "volumes" feature, which requires version docker 1.10 or greater (the default is 1.9 for ubuntu 14.04). To utilize docker 1.10 as a replacement, use the following:
+```
+machine:
+  pre:
+    - docker version && sudo stop docker && curl -sSL https://s3.amazonaws.com/circle-downloads/install-circleci-docker.sh | bash -s -- 1.10.0 && sudo start docker && docker version
+```
+A common problem, however, when using version docker 1.10 is that the layering hierararchy is not preserved across pulls, even when using the save/load caching mechanisms described above. ([Docker](Link.TK) and [CircleCI](Link.TK) are both aware of this limitation.) As a workaround, you can use the default docker version to build your images and then upgrade to docker v.1.10 as the last step of your dependency script. For example:
```

Comment:
TODO (TK): Need to provide links to the relevant forum entries and/or GitHub issues.